src/main/java/seedu/address/model/person/Score.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's score in JABPro.
+ */
+public class Score implements Comparable<Score> {
+    public static final String MESSAGE_CONSTRAINTS = "Score should be a non-negative integer";
```

Comment:
Is there a reason why we're not enforcing a constraint on what the maximum score should be?

Reply:
Multiple stages of interview may constitute more than a set score. This is the basic structure of score 

Also different companies might want to set a different max score so its not really flexible to include it now